### PR TITLE
add toyota into common.js

### PR DESCRIPTION
**Pull request by ZenVega**
another try. I nuked everything down. man, this started out so chill, and then I created a monster.
I believe it should work now. thanks for the patience. and if u would add a hacktoberfest label I would appreciate it.
by the way, I had to install git-lfs. Don't know if that was on purpose

### CODE REVIEW

**Review by Bjorn248 — CHANGES_REQUESTED** [review]
Seems like there's an empty model here. I will consolidate a lot of the duplicate models you have listed here, so don't worry about that.

File: common.js [hunk 1]
```diff
@@ -5601,6 +5601,343 @@ const allSoloCars = {
...
+      '1972': ['fsp'],
+      '1973': ['fsp'],
+      '1974': ['fsp'],
+      '1975': ['fsp'],
+      '1976': ['fsp'],
+      '1977': ['fsp'],
+      '1978': ['fsp'],
+      '1979': ['fsp'],
+    },
+    'Corolla (1600 & 1800,RWD)': {
```
[the remark this answers is not in the document]
Comment (ZenVega) [reply common.js:5755]: ah, nope. that one got lost

**Comment by Bjorn248** [comment]
Thanks!

**Comment by ZenVega** [comment]
man, thanks for the patience. good luck with your app

**Pull request merged**